packages/block-editor/src/components/alignment-toolbar/index.js

```diff
@@ -38,7 +38,7 @@ export function AlignmentToolbar( props ) {
 		value,
 		onChange,
 		alignmentControls = DEFAULT_ALIGNMENT_CONTROLS,
-		label = __( 'Change text alignment' ),
+		label = __( 'Align' ),
```

Comment:
I feel a bit nervous about changing the label to something so succinct, are we confident it's informative enough for screen reader users?

This one in particular because it's using the same label as block alignment.

Reply:
Hmm, might be worth adding the full text as a tooltip here.

Reply:
Came here to say something similar to Dan. As part of fixing a regression after this PR got merged, it seems that we [rely on now single-word label](https://github.com/WordPress/gutenberg/pull/28286/files#diff-4749eb3277e22380531b254004b270c0d603cc201321292f13572abf7509a6f0R27) to find the button in E2E tests and tap it on native mobile. My small concern is whether that single word label is unique enough for confident search of it. I don't think it's an issue atm, but wanted to share the concern.

Reply:
We can always resort to a combination of classnames, or other attributes, and labels to find the item. E2e tests shouldn't determine UI design 😅